src/radsig_io.jl

```diff
@@ -5,7 +5,7 @@ function to_table(x::AbstractVector{<:RDWaveform})
     TypedTables.Table(
         t0 = first.(x.time),
         dt = step.(x.time),
-        values = x.value
+        values = x.signal
```

Comment:
Question is: how do we want to deal with this?
Right now, if we convert a `Vector` of `RDWaveform` to a `Table`, the column will be called `values`.
Not changing this allows to read in previously saved `Tables` but then deviates from the new naming, see 
https://github.com/legend-exp/LegendHDF5IO.jl/blob/b89777180aa7753b37fe75d8a5caf63ef37e407e/src/radsig_io.jl#L19-L24

Reply:
Try this:

```julia
@static if hasfield(RDWaveform, :signal)
    _get_wf_signal(x) = x.signal
else
    _get_wf_signal(x) = x.value
end

...

TypedTables.Table(
        t0 = first.(x.time),
        dt = step.(x.time),
        values = _get_wf_signal(x)
```

Reply:
Do you want me to implement this?
If we keep the naming of the tables for compatibility, I think this PR can be merged right away
(and then the right version of LegendHDF5IO is loaded depending on the version of RadiationDetectorSignals)